h/services/job_queue.py

```diff
@@ -34,6 +37,7 @@ def get(self, name, limit):
     def delete(self, jobs):
         for job in jobs:
             self._db.delete(job)
+            log.info("Deleted job: %s", job)
```

Comment:
Let's log whenever deleting a job from the job table.

FIXME: This should be `%r` for `__repr__()` not `__str__()`.